packages/common/src/GRPCClient.ts

```diff
@@ -309,35 +309,68 @@ export class ConcordiumGRPCClient {
         transaction: v1.AccountTransaction,
         signature: v1.AccountTransactionSignature
     ): Promise<HexString> {
-        const rawPayload = serializeAccountTransactionPayload(transaction);
-        const transactionSignature: v2.AccountTransactionSignature =
-            translate.accountTransactionSignatureToV2(signature);
-
-        // Energy cost
         const accountTransactionHandler = getAccountTransactionHandler(
             transaction.type
         );
+
+        const rawPayload = serializeAccountTransactionPayload(transaction);
+
+        // Energy cost
         const baseEnergyCost = accountTransactionHandler.getBaseEnergyCost(
             transaction.payload
         );
+
         const energyCost = calculateEnergyCost(
             countSignatures(signature),
             BigInt(rawPayload.length),
             baseEnergyCost
         );
 
+        return this.sendRawAccountTransaction(
+            transaction.header,
+            energyCost,
+            rawPayload,
+            signature
+        );
+    }
+
+    /**
+     * Sends an account transaction, with an already serialized payload, to the node to be
+     * put in a block on the chain.
+     *
+     * Note that a transaction can still fail even if it was accepted by the node.
+     * To keep track of the transaction use getTransactionStatus.
+     *
+     * Note that { @link ConcordiumGRPCClient.sendAccountTransaction } is the recommended
+     * method to send account transactions, as this requires the caller to serialize the payload themselves.
```

Comment:
It sounds like `sendAccountTransaction` is the reccommended way of sending transactions, as it requires the user to hand serialize payloads 🧑‍🏭 

```suggestion
     * In general, { @link ConcordiumGRPCClient.sendAccountTransaction } is the recommended
     * method to send account transactions, as this does not require the caller to serialize the payload themselves.
```